code/src/java/pcgen/cdom/processor/ChangeArmorType.java

```diff
@@ -148,10 +136,6 @@ public boolean equals(Object obj)
 			return false;
 		}
 		ChangeArmorType other = (ChangeArmorType) obj;
-		if (result == null)
```

Comment:
I removed many of the docs. Reading over the rest of the code and history this was modified in cd0f00c2843b4a8b978b6e8479b4690260544f7e but there was related changes prior to that which actually caused the null to be useless. I suspect just deleting the docs is correct.